lib/Index/Index.cpp

```diff
@@ -899,6 +899,13 @@ bool IndexSwiftASTWalker::startEntityDecl(ValueDecl *D) {
     }
   }
 
+  if (auto EED = dyn_cast<EnumElementDecl>(D)) {
+     EnumDecl *ED = EED->getParentEnum();
+     for (auto *accessAttr : ED->getAttrs().getAttributes<AccessControlAttr, true>()) {
+       EED->getAttrs().add((DeclAttribute *)accessAttr);
+     }
+  }
```

Comment:
I initially thought this would be possible by changing `SymbolInfo`. Since that's not the case I don't believe this is the best place for this change, but I wasn't sure where else to put it.